src/js/cityDetails.js

```diff
@@ -0,0 +1,40 @@
+/* global document */
+
+const setUpDetails = (markerGroup, data) => {
+  const cityDetailsElement = document.querySelector(".city-details-popup");
+
+  markerGroup.on("click", function (e) {
+    const cityState = e.sourceTarget.getTooltip().getContent();
+
+    cityInfo = createCityDetailsText(data, cityState);
+
+    document.querySelector(".city-details-text").innerHTML = cityInfo;
+
+    cityDetailsElement.style.display = "block";
+  });
+
+  // Note that the close element will only render when the about text popup is rendered.
+  // So, it only ever makes sense for a click to close.
+  document
+    .querySelector(".city-details-popup-close-icon")
+    .addEventListener("click", () => {
+      cityDetailsElement.style.display = "none";
+    });
+};
+
+const createCityDetailsText = (data, cityState) => {
+  const cityData = data[cityState];
+
+  cityInfo = `<h2><a href="${cityData["citation_url"]}">${cityState}</h2>
```

Comment:
No need for the blank line. Also you can directly return the right hand side -- no need to create the `cityInfo` variable.